underlay/src/main/java/bio/terra/tanagra/query/bigquery/BQQueryRunner.java

```diff
@@ -53,7 +54,7 @@ public ListQueryResult run(ListQueryRequest listQueryRequest) {
     // Build the SQL query.
     Instant queryInstant =
         listQueryRequest.getPageMarker() == null
-            ? Instant.now()
+            ? instantNowMicros()
```

Comment:
I think it's better to truncate or round to microseconds at the point we pass the timestamp value to BQ. Doing it here isn't really confined to the BQ code, because this `queryInstant` is part of the return value of this function as well. Can you truncate in `BQExecutor.toQueryParameterValue` instead? That would also guarantee that we truncate in any instances where the timestamp is turned into a literal. I will push the example code to a branch.

Reply:
Closing this since the change is merge.